app/layout.tsx

```diff
@@ -1,24 +1,27 @@
-import Head from 'next/head'; // Import the Head component
-import { Montserrat } from "next/font/google";
-
-import './globals.css'
-
-const montserrat = Montserrat({ subsets: ["latin"] });
+import type { Metadata } from "next";
+import "./globals.css";
+import Header from "@/components/Header";
 
 export const metadata: Metadata = {
-  title: 'Modular UI',
-  description: 'Craft beautiful, accessible, and responsive Flutter UIs with a Modular-UI Design-inspired component library',
+  icons: {
+    icon: "/icon.ico",
+  },
+  title: "Modular UI",
+  description:
+    "Craft beautiful, accessible, and responsive Flutter UIs with a Modular-UI Design-inspired component library",
 };
 
-export default function RootLayout({ children }: { children: React.ReactNode }) {
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
   return (
     <html lang="en">
-      <Head>
-        <title>{metadata.title}</title>
-        <meta name="description" content={metadata.description} />
-        <link rel="icon" href="/favicon.ico" /> {/* Add the favicon link here */}
-      </Head>
```

Comment:
Please add a Head section to provide `meta` information because it helps to tell the browser about the website and ultimately we get good SEO ranking too.

Reply:
Warning: You're using `next/head` inside the `app` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead for more details.
<img width="767" alt="Screenshot_1" src="https://github.com/opxica/modular-ui-website/assets/150684645/4219946e-b4cf-4170-ac92-8ea277a1c626">

Reply:
It's not `<Head>` it's `<head>`. It's considering as components right now but it's just HTML Tag...